src/ray/common/scheduling/scheduling_alg.cc

```diff
@@ -0,0 +1,204 @@
+#include "scheduling_alg.h"
+#include <iostream>
+#include <stdlib.h>
+#include <chrono>
+
+using namespace std;
+
+ClusterResources::ClusterResources(int64_t local_node_id, const NodeResources &node_resources) {
+  local_node_id_ = local_node_id;
+  add(local_node_id, node_resources);
+}
+
+void ClusterResources::add(int64_t node_id, const NodeResources &node_resources) {
```

Comment:
```suggestion
void ClusterResources::AddNode(int64_t node_id, const NodeResources &node_resources) {
```
Function names should start with upper case and be MixedCase
https://google.github.io/styleguide/cppguide.html#Function_Names

Reply:
also prefer to be specific over concise - in this case `AddNode` makes things clearer